crates/oauth2-types/src/errors.rs

```diff
@@ -30,7 +30,7 @@ pub struct ClientError {
     pub error: ClientErrorCode,
 
     /// A human-readable description of the error.
-    pub error_description: Cow<'static, str>,
+    pub error_description: Option<Cow<'static, str>>,
```

Comment:
This removes the field instead of having it set as `null` when serializing:

```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    pub error_description: Option<Cow<'static, str>>,
```

Reply:
Ah yeah, good point I didn't think about serialising. Updated.